testing/vcs/test_vcs_dump_json.json

```diff
@@ -220,8 +224,8 @@
    "datawc_x2": 1e+20, 
    "datawc_y1": 1e+20, 
    "datawc_y2": 1e+20, 
-   "line": null, 
    "linecolor": null, 
+   "linetype": null,
```

Comment:
oh yes that is why we need to keep backward compatibilty for a while, all the json file have `line` in them. That will break them all.